geonode/base/models.py

```diff
@@ -624,8 +625,8 @@ def set_latlon_bounds(self, box):
         Set the four bounds in lat lon projection
         """
         self.bbox_x0 = box[0]
-        self.bbox_x1 = box[1]
-        self.bbox_y0 = box[2]
+        self.bbox_y0 = box[1]
+        self.bbox_x1 = box[2]
```

Comment:
This is like a very consistent inconsistency 😟 and it's been around since 2012.
The problem is as follow:

If you see bbox property from above (in the same file)
the format is: [x0,y0,x1,y1] which is probably the standard format, CMIIW.

However, set_latlon_bounds assumes the format: [x0,x1,y0,y1]

Some bbox calculation in maps creation seems wrong because it takes layer.bbox (which inherits from base), but comparing it as if [x0,x1,y0,y1] to get the largest extent of multiple layers. Then it uses this method to set the bbox. Which is swapped.